<div class="panel-heading" id="headingOne">
    <h4>Instructions
        <button id = 'instr-toggle-button' class="btn btn-default btn-sm" data-toggle="collapse" data-target="#collapseOne">
        <span class="more-less glyphicon glyphicon-plus"></span>
    </button></h4>
</div>
<div id="collapseOne" class="panel-collapse collapse">
    <p>Welcome to <b>Baseline CompaRe</b>. This tool is designed to help with the
    analysis of E8.5-10.5 trancriptomics data, particularly in cases where
    the embryos of interest are developmentally delayed.
    We have produced RNA-seq data for wild-type embryos for stages from
    4 to 36 somites to form a baseline for comparison to experimental samples.
    This allows us to prioritise genes that are more likely to be differentially
    expressed due to the condition of interest, rather than due to developmental delay.</p>
    <p>To get started you need pre-computed count data from your RNA-seq data.</p>
    <h3 id='file_formats'>File Formats</h3>
    <p><b><i>EXAMPLE FILES - Example files can be downloaded at link</i></b></p>
    <p>Two files are required:</p>
    <h4>Sample File</h4>
    Information on the samples in the experiment.
    <table class = "table">
        <thead>
            <tr><td></td><td>condition</td><td>stage</td><td>sex</td><td>sample_names</td></tr>
        </thead>
        <tbody>
            <tr><td>Sample_1</td><td>sib</td><td>24somites</td><td>F</td><td>Brd2_wt_1</td></tr>
            <tr><td>Sample_2</td><td>sib</td><td>22somites</td><td>F</td><td>Brd2_wt_2</td></tr>
            <tr><td>Sample_3</td><td>sib</td><td>26somites</td><td>F</td><td>Brd2_het_1</td></tr>
            <tr><td>Sample_4</td><td>hom</td><td>14somites</td><td>F</td><td>Brd2_hom_1</td></tr>
        </tbody>
    </table>
    <h5>Columns</h5>
    <p>This should be a tab-separated file containing the following columns:</p>
    <ol>
        <li><p><b>sample identifiers</b> - The first column must be sample identifiers.
    They should match the sample identifiers in the Count file.</p></li>
        <li><p><b>condition/treatment</b> - The condition or treatment being investigated.
    e.g. genotype or drug treatment</p></li>
        <li><p><b>stage</b> - The developmental stage in somites</p></li>
        <li><p><b>sex</b> - sex of the embryos F/M. This is optional</p></li>
        <li><p><b>sample_names</b> - An optional column of samples names can be included</p></li>
    </ol>
    <h4>Count File</h4>
    A file of counts for Ensembl genes for each sample
    <table class = "table">
        <thead>
            <tr><td>Gene.ID</td><td>Sample_1 count</td><td>Sample_2 count</td><td>Sample_3 count</td><td>Sample_4 count</td></tr>
        </thead>
        <tbody>
            <tr><td>ENSMUSG00000000001</td><td>4192</td><td>5151</td><td>4670</td><td>2014</td></tr>
            <tr><td>ENSMUSG00000000003</td><td>0</td><td>0</td><td>0</td><td>0</td></tr>
            <tr><td>ENSMUSG00000000028</td><td>1143</td><td>1398</td><td>1201</td><td>1089</td></tr>
            <tr><td>ENSMUSG00000000031</td><td>44807</td><td>51227</td><td>45521</td><td>44186</td></tr>
        </tbody>
    </table>
    <h5>Columns</h5>
    <p>This should be a tab-separated file. A column of Ensembl Gene identifiers
    named <b>Gene.ID</b> is required. The remaining columns should be the counts
    for the individual samples. The column names should be of the form
    <i>sample_id</i> count</p>
    <p>Once the data is loaded, the <b>Options</b> panel with change based on
    the column names.</p>
    <h5>Condition:</h5>
    <p>Select the column name from you samples file that
    describes the condition of each embryo. Currently, accepted entries in the
    condition column are 'wt', 'het', 'hom', 'sib' and 'mut'. The DESeq analysis
    combines wt and het entries into a sib category.</p>
    <h5>Sex:</h5>
    <p>If your samples file contains information on the sex of the embryos this
    can be included in the analysis to account for differences driven by
    differences in sex. Use the checkbox to include sex in the analysis or not.
    If the checkbox is ticked, please choose the correct column name from the
    radio buttons.</p>
    <p>Once the options are set correctly, click "Analyse Data"</p>
</div>